test/e2e/ingress.go

```diff
@@ -673,5 +681,21 @@ var _____ = framework.KubeDescribe("Ingress tests simple NLB", func() {
 		s, err := getBody(resp)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(s).To(Equal(backendContent))
+
+		By("Checking request X-Forwarded-* headers")
+		req, err = http.NewRequest("GET", "https://"+hostName+"/", nil)
+		Expect(err).NotTo(HaveOccurred())
+		resp, err = waitForResponseReturnResponse(req, 10*time.Second, isSuccess, false)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(resp.Header.Get("Y-X-Forwarded-For")).NotTo(Equal(""))
+		Expect(resp.Header.Get("Y-X-Forwarded-Port")).To(Equal("443"))
```

Comment:
This here returns 
```
 ESC[91mESC[1m• Failure [163.716 seconds]ESC[0m
 [k8s.io] Ingress tests simple NLB
 ESC[90m/workspace/test/e2e/e2e_modules/kubernetes/test/e2e/framework/framework.go:624ESC[0m
   ESC[91mESC[1mShould create simple NLB ingress [Ingress] [Zalando] [It]ESC[0m
   ESC[90m/workspace/test/e2e/ingress.go:606ESC[0m

   ESC[91mSep 28 16:52:35.218: Expected
       <string>: ${request.header.X-Forwarded-Port}            <------- what?
   to equal
       <string>: 443ESC[0m

   /workspace/test/e2e/ingress.go:691
```

Reply:
e2e tests use outdated skipper https://github.com/zalando-incubator/kubernetes-on-aws/blob/9845fe22ca83bf164b8460dc5f8e85a184f111bb/test/e2e/util.go#L401